src/make_table_example.F90

```diff
@@ -157,8 +157,8 @@ program make_table_example
   Imax_logtemp = log10(150.0d0)
   Imin_logeta = log10(0.1d0)
   Imax_logeta = log10(100.0d0)
-  Imin_logn_N = 35.0d0 
-  Imax_logn_N = 37.0d0
+  Imin_logn_N = 20.0d0 
+  Imax_logn_N = 40.0d0
```

Comment:
This is too big now (only 4 points per decade of density).  Can you choose these based on the density limits of the table? Something like Imax_logn_N = log ( 10**max_logrho / amu ) and Imin_logn_N = log ( 10**min_logrho / amu )

Reply:
I can try to have something like min_nN = (min_Ye * min_logrho /m_amu)   &    max_nN = ((1-min_Ye) * max_logrho /m_amu) 

Sometimes the code goes below that limit, so a condition putting the rates to 0 when off-limit should be added then.